### PR TITLE
[#2 Action dropdown service row] Add action dropdown on service rows

### PR DESCRIPTION
**Pull request by kennyt**
DEPENDS ON #681  (Merged)

px

jash wanted the icon to be a gear with a caret

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @orlandohohmeier

**Comment by orlandohohmeier** [comment]
Needs a rebase... Besides that it looks good to me. Great job! 💯 ✨

File: src/js/components/ServicesTable.js [hunk 1]
```diff
@@ -74,10 +153,135 @@ var ServicesTable = React.createClass({
...
+    );
+  },
+
+  getDestroyConfirmDialog: function () {
+    const {state} = this;
+    let service = state.serviceToChange;
+
+    let message = (
+      <div className="container-pod flush-top container-pod-short-bottom">
+        <h2 className="text-danger text-align-center flush-top">Destroy Service</h2>
```
Comment (pierlo-upitup) [inline src/js/components/ServicesTable.js]: We need to check if this is a group (`service instanceof ServiceTreee` should do) and ask to `Destroy Group` in that case.

Comment (pierlo-upitup) [inline src/js/components/ServicesTable.js]: Same goes for all other actions ;)

Comment (kennyt) [inline src/js/components/ServicesTable.js]: done

**Comment by jfurrow** [comment]
Can we wrap the group/service name in `<span className="emphasize">`? That will help match our "Create Group" modal:

**Comment by kennyt** [comment]
@jfurrow done

**Comment by jfurrow** [comment]
LGTM

**Pull request merged**